### ISSUE TITLE
Add Rspec

### ISSUE THREAD

**Issue by denispasin** [THP-2-0/THP2_back#2]
### Must have:
- [ ] rspec-rails
- [ ] basic `.rspec` config must include rails_helper by default

**Comment by C83** [comment]
I'am doing the third point. 
My .rspec file contains `--require spec_helper`
The second line of rails_helper is `require spec_helper`
If I add `--require rails_helper` in .rspec file, the spec_helper will be loaded twice ?

**Comment by denispasin** [comment]
Nope. That's a mechanism of ruby. If something is already required it won't be required again.
